packages/components/src/date-time/style.scss

```diff
@@ -1,126 +1,207 @@
-$datepicker__border-radius: 0;
-$datepicker__day-margin: 0.166rem;
-$datepicker__font-size: 13px;
-$datepicker__font-family: inherit;
-$datepicker__item-size: 28px;
-$datepicker__margin: 0.4rem;
-$datepicker__navigation-size: 6px;
-$datepicker__triangle-size: 6px;
-
-$datepicker__background-color: $light-gray-300;
-$datepicker__text-color: $dark-gray-500;
-$datepicker__header-color: $black;
-$datepicker__muted-color: #ccc;
-$datepicker__navigation-disabled-color: lighten($datepicker__muted-color, 10%);
-$datepicker__border-color: $light-gray-500;
-
-// Lerna hoist packages, so can't reference with ~
-@import "../../../../node_modules/react-datepicker/src/stylesheets/datepicker";
-
-.react-datepicker__month-container {
-	float: none;
-}
-
-.react-datepicker-time__header,
-.react-datepicker__current-month {
-	font-size: $datepicker__font-size;
-}
+.components-datetime__date {
+	min-height: 236px;
+	border-top: 1px solid $light-gray-500;
+	margin-left: -$item-spacing;
+	margin-right: -$item-spacing;
+
+	.components-datetime__date__aria-description {
+		position: absolute;
+		top: 0;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		background-color: $white;
+		z-index: 100;
+		padding: $item-spacing;
+
+		&--state-invisible {
+			display: none;
+		}
+	}
 
-.react-datepicker__navigation {
-	top: 12px;
+	.components-datetime__date__help-button {
+		margin-top: 0.5em;
+		margin-right: $item-spacing;
+		float: right;
```

Comment:
I'm always suspect of using floats, but I guess it's fewer rules than the alternate, which here would be `display: block; margin-left: auto;`. Still, the box model is more predictable not using floats, so maybe this could be changed? 🤷‍♂️

Reply:
Yeah… I was thinking about absolute positioning and I guess I ended up deciding on using `float` as a compromise.

I was also a bit concerned with how the bounding box would change depending on screen size and localisation.

Let me know if you have a good solution.